config/default/test/.jshintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "esversion": 6,
```

Comment:
Safe to delete this file since jshint is no longer used

Reply:
Thanks, removed it.